spec/fixtures/validation_mailer/validation_email_spec.rb

```diff
@@ -0,0 +1,21 @@
+
+require 'rails_helper'
```

Comment:
still working through testing properly. documentation has been proving hard to track down